app/controllers/project_users_controller.rb

```diff
@@ -52,6 +51,17 @@ def respond
     end
   end
 
+  def flash_error
+    error_message = @project_user.errors.messages[:user_id]
+    message = if error_message.present?
```

Comment:
anh nghĩ cái chị muốn làm là nó hiển thị message kiểu này chứ không phải flash, cái này confirm lại với chị nhá
![image](https://github.com/awesome-academy/project_management/assets/137058348/7da2bc86-8cd1-4988-b334-7c9bf2c98ba8)

Reply:
Cái message này vẫn hiện a ạ, cái fix của e là lúc đầu flash nó hiển thị sai, nó thêm chữ "danger" ở cuối câu.
Em làm như trên để flash hiển thị chính xác lỗi "trùng người dùng" ấy a.

Reply:
dùng toán tử 3 `ngôi viết trên 1 dòng dc k